google-cloud-clients/google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingImpl.java

```diff
@@ -142,7 +143,8 @@ public Severity getFlushSeverity() {
           public O apply(I i) {
             return function.apply(i);
           }
-        });
+        },
+        MoreExecutors.directExecutor());
```

Comment:
Please use static import for `MoreExecutors.directExecutor()`.

Reply:
done